evap/static/ts/src/auto-form-saver.ts

```diff
@@ -0,0 +1,198 @@
+/**
+Derived from https://github.com/simsalabim/sisyphus, originally distributed under the following license:
```

Comment:
We still mention sisyphus in our `LICENSE.md` file, I suppose we should remove it then? @richardebeling